internal/utils/grpc.go

```diff
@@ -16,10 +19,48 @@ func CloseClientConnection(connection *grpc.ClientConn) {
 }
 
 func GrpcDialWithInsecure(serviceName string, serviceURL string) (*grpc.ClientConn, error) {
-	cc, err := grpc.Dial(serviceURL, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	kacp := keepalive.ClientParameters{
+		Time:                60 * time.Second, // ping the server every ~60 seconds.
+		Timeout:             20 * time.Minute, // wait up to 20 minutes for a packet to be acknowledged.
```

Comment:
Isn't this value a bit high?

Reply:
It may be, we can halve it to 10 minutes, what number would you suggest ?

Reply:
[gRPC defaults](https://grpc.io/docs/guides/keepalive/#keepalive-configuration-specification) seems to be 20s for both client and server.
![image](https://github.com/berops/claudie/assets/42786030/50c518a1-b92c-45ae-b29f-f426418bb2ea)

Reply:
I think 20s is a bit low, I would set it to 5 mins.

Reply:
Sure, let's set it to 5min then